README.md

```diff
@@ -1,7 +1,9 @@
 # OpenEBS CStor CSI Driver
```

Comment:
n_: @prateekpandey14 -- let us change from CStor to "cStor" everywhere.

Reply:
done